src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -62,13 +77,16 @@ public MigrateRepoCommand(OctoLogger log, ITargetGithubApiFactory targetGithubAp
             AddOption(sourceRepo);
             AddOption(githubTargetOrg);
             AddOption(targetRepo);
+            AddOption(targetApiUrl);
+            AddOption(metadataArchiveUrl);
+            AddOption(gitArchiveUrl);
             AddOption(ssh);
             AddOption(verbose);
 
-            Handler = CommandHandler.Create<string, string, string, string, string, string, bool, bool>(Invoke);
+            Handler = CommandHandler.Create<string, string, string, string, string, string, string, string, string, bool, bool>(Invoke);
         }
 
-        public async Task Invoke(string githubSourceOrg, string adoSourceOrg, string adoTeamProject, string sourceRepo, string githubTargetOrg, string targetRepo, bool ssh = false, bool verbose = false)
+        public async Task Invoke(string githubSourceOrg, string adoSourceOrg, string adoTeamProject, string sourceRepo, string githubTargetOrg, string targetRepo, string targetApiUrl, string metadataArchiveUrl = "", string gitArchiveUrl = "", bool ssh = false, bool verbose = false)
```

Comment:
Not sure how we can make this better but the number of args are getting out of hand and it usually is a code smell.

Reply:
We could update this in a followup